<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegToolBox</class>
 <widget class="QMainWindow" name="RegToolBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="template_view">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/icons/axial.xpm</normaloff>:/icons/icons/axial.xpm</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Zoom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="main_zoom">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>50</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="dis_spacing">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Vector Field x1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Vector Field x2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Vector Field x4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Vector Field x8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Vector Field x16</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGraphicsView" name="main_view"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QSlider" name="slice_pos">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Open">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Save_Warpped_Image"/>
   </widget>
   <addaction name="menu_Open"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Image</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Template Image</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QGraphicsView" name="It_view"/>
        </item>
        <item>
         <widget class="QPushButton" name="OpenTemplate">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Subject Image</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QGraphicsView" name="I_view"/>
        </item>
        <item>
         <widget class="QPushButton" name="OpenSubject">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>Method</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Linear Registration</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="linear_type">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>Rigid Body</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Affine</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Cost Fun</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cost_func">
          <item>
           <property name="text">
            <string>Correlation</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mutual Info</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Nonlinear Registration</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Method</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>SPM norm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DMDM</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Speed:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="speed">
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>3.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.450000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Detail (mm)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="resolution">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>0.500000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>2.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="reg_type">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>Linear Reg</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Nonlinear Reg</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="run_reg">
         <property name="text">
          <string>Start</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/icons/run.xpm</normaloff>:/icons/icons/run.xpm</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Save_Warpped_Image">
   <property name="text">
    <string>&amp;Save Warpped Image...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
